src/machine.rs

```diff
@@ -34,7 +35,6 @@ pub const STACK_SIZE: u64 = 16 * PAGE_SIZE; // whatever
 pub const NUM_CPUS: u64 = 1;
 
 /// Extra data stored with each stack frame
-#[derive(Debug)]
```

Comment:
Losing `Debug` isn't great, this could make debugging in the future more annoying.
Can you add `impl Debug for DetachedTiming` on the measureme side? Actually getting that shipped doesn't have to block this PR, but it'd be better to at least have a plan to get this `Debug` here back.

Reply:
I've added a manual `Debug` impl that omits `DetachedTiming`

Reply:
That also works for now, thanks.
(But I'd still encourage measureme to `impl Debug` for all its public types.)